modules/ggl-uri/src/gghttp_uri.c

```diff
@@ -136,3 +138,283 @@ GglError gg_uri_parse(GglArena *arena, GglBuffer uri, GglUriInfo *info) {
 
     return GGL_ERR_OK;
 }
+
+static GglError find_docker_uri_separators(
+    GglBuffer uri,
+    size_t slashes[2],
```

Comment:
Why not get a pointer instead of an array? This doesn't help the compiler. If you pass in a pointer which points to only one valid location (instead of 2), compiler won't catch the bug.

Reply:
`size_t slashes[static 2]` would require at least 2. Used for compiler options; may warn.

Without the static, its ignored by compiler and just for humans.